custom_components/daikin_onecta/daikin_api.py

```
@@ -69,93 +70,72 @@ async def async_get_access_token(self) -> str:
+                            )
+                        if method == "GET":
+                            return []
+                        else:
+                            return False
+                    elif resp.status == 204:
+                        self._last_patch_call = datetime.now()
+                        return True
+
+                    _LOGGER.debug("BEARER RESPONSE CODE: %s LIMIT: %s", resp.status, self.rate_limits)
+
             except Exception as e:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)